site/public/css/colors.css

```diff
@@ -14,6 +14,7 @@
     --focus-warning-brown: #ad7c37;
     --hover-warning-brown: #9c5b00;
     --active-warning-brown: #b58a4e;
+    --default-green: #018201;
```

Comment:
This change seems reasonable to me but I'd like to get @jenaybarela's approval before adding a new color.  This is related to the corresponding change on line 322.